broker-daemon/models/block-order.js

```diff
@@ -476,6 +438,67 @@ class BlockOrder {
 
     return BlockOrder.fromStorage(blockOrderId, value)
   }
+
+  /**
+   * @param {Array<Order>} orders
+   * @returns {Object} res
+   * @returns {Big} res.inbound
+   * @returns {Big} res.outbound
+   */
+  static async activeAmountsForOrders (orders) {
```

Comment:
maybe this belongs somewhere else, like `BlockOrderWorker`, or `OrderStateMachine`. It doesn't have much to do with the `BlockOrder`.